### ISSUE TITLE
Error loading... No Gift pass 

### ISSUE THREAD

**Issue by elRadix** [Fllorent0D/ha-custom-component-pluxee-be#1]
First of all thank you for all your effort!!

Below is de error i am receiving while using the correct login details
I don't have a gift pass. 

```
Logger: homeassistant.components.sensor
Source: custom_components/sodexo/api.py:58
Integration: Sensor (documentation, issues)
First occurred: 12:45:51 (3 occurrences)
Last logged: 12:45:51

sodexo: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 477, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 702, in async_device_update
    await task
  File "/config/custom_components/sodexo/sensor.py", line 123, in async_update
    account = await api.getAccountDetails()
  File "/config/custom_components/sodexo/api.py", line 58, in getAccountDetails
    gift = float(soup.select_one(GIFT_PASS_SELECTOR).text.replace(' €', ''))
AttributeError: 'NoneType' object has no attribute 'text'
```

This why I get using the sodexo interface

![SmartSelect_20220815-125234_Chrome](https://user-images.githubusercontent.com/6258894/184623136-692ea26a-fea5-42d7-9aba-14a9afec2b96.jpg)

**Comment by Fllorent0D** [comment]
I knew that I add to check if the tag was existing before getting the value... 
Fixed with d2985a192da9ffeb8e31c92a6b2caa9167c37949

Can you confirm if it's working with the fix?

**Comment by elRadix** [comment]
It's fixed now,  thank you for your quick response!
![SmartSelect_20220815-195452_Home Assistant](https://user-images.githubusercontent.com/6258894/184689288-adfaa961-8c39-44b3-9408-babee765c55f.jpg)
